pkg/swap/swap.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/sh
+
+set -e
+
+while [ $# -gt 1 ]; do
+	key="$1"
+
+	case $key in
+		--debug)
```

Comment:
Atom seems to be doing indentation inconsistently. Mostly tabs, sometimes spaces.